core/src/main/codegen/templates/Parser.jj

```diff
@@ -5903,13 +5904,21 @@ SqlTypeNameSpec SqlTypeName1(Span s) :
     |
         <BOOLEAN> { s.add(this); sqlTypeName = SqlTypeName.BOOLEAN; }
     |
-        ( <INTEGER> | <INT> ) { s.add(this); sqlTypeName = SqlTypeName.INTEGER; }
+        ( <INTEGER> | <INT> ) ( <UNSIGNED> { unsigned = true; })? {
```

Comment:
Considering that not all databases support `UNSIGNED` types, can we add `conformance.allowUnsignedType()` to perform checks?